package.json

```diff
@@ -140,7 +140,7 @@
     "semver": "^5.6.0",
     "static-site-generator-webpack-plugin": "github:skyscanner/static-site-generator-webpack-plugin#7bac616",
     "style-loader": "^0.23.1",
-    "stylelint": "^9.8.0",
+    "stylelint": "^9.9.0",
```

Comment:
Bunch of new things - build still passes 👍, [changelog](https://github.com/stylelint/stylelint/blob/master/CHANGELOG.md#990)